_posts/2023-10-24-filesystem-integrity-elytron-tool.adoc

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: 'Adding Support for Integrity Checking to an Existing 
+Filesystem Realm'
+date: 2023-10-24
+tags: filesystem integrity filesystem-realm
+synopsis: How to add Integrity checking to an existing filesystem realm using the Elytron tool. 
+author: prarthonapaul
+---
+
+:toc: macro
+:toc-title:
+
+WildFly 27 included the ability to add integrity on FileSystem realms. This uses an asymmetric keyPair that has a private and public key to secure the filesystem realm. In the filesystem realm, each identity will be signed using the private key. This makes it possible to verify that identities in the realm haven't been tampered with.
```

Comment:
s/integrity on FileSystem realms/integrity checking support to FileSystem Realms

Reply:
s/keyPair/key pair

Reply:
Fixed!